### ISSUE TITLE
Commenting Feature R&D

### ISSUE THREAD

**Issue by tpowell** [IWT-Book/Book-Master#3]
Add in a right bar icon or bottom icon that upon click that allows a user to add comments.  Gitbook itself may support this at the bottom of the page so we may not have to do much

https://help.gitbook.com/books/how-can-i-allow-comments-on-my-book.html

Plugin that is official

https://plugins.gitbook.com/plugin/comment

**Comment by raafatdavid** [comment]
Yes I was able to gather more information about the comment things,  and I documented all the plugins that might be useful for this task under the Useful Plugins. I'll need to test them and see which ones will fit our needs.
